util/platform_symbols/unix-symbols.txt

```diff
@@ -142,6 +142,7 @@ strdup
 strlen
 strncmp
 strncpy
+strpbrk
```

Comment:
It needs to be added also to the windows-symbols.txt

Reply:
Done